docs/demo_guide.md

```diff
@@ -241,25 +241,31 @@ DEMO B: Running an Archive Rule
     the rule is enabled. This will enable you to have an **automated retention
     policy** in Relativity.
     
-    > **NOTE:** For this demo retention has be purposely set to 0 hours, which means that all of the `non-alerted` data will be
```

Comment:
Retention is no longer set to zero? Curious why we removed the note? @cpvanfossan

Reply:
Never mind, I see another bullet with this info

Reply:
This is the issue I was having with hopper. Every time I deploy the image, the retention policy is being set to 168 hours.

Reply:
This is a bit suspicious. What resets the number? If template has it set to zero, it should stay as zero.

Something to double check